src/sv-pipeline/scripts/SplitVariants.py

```diff
@@ -0,0 +1,72 @@
+#!/bin/python
+import argparse
+
+
+def process_bed_file(input_bed, N, bca=True):
+    condition_prefixes = {
+        'gt5kb': {
+            'condition': lambda line: (line[4] == 'DEL' or line[4] == 'DUP') and (int(line[2]) - int(line[1]) >= 5000)},
```

Comment:
Ok. This was done, and also for the other numbers referring to start and end position to calculate the distances.

Reply:
Thanks. Small style thing - since these are constants they should be in all caps. I would probably call end_pos END_FIELD and start_pos START_FIELD as well so that it doesn't seem like the actual coordinates are 1 and 2

Reply:
Ok. I did that initially but PyCharm did not accept that. They are reverted back to all caps.